va/validation-authority.go

```diff
@@ -487,6 +487,11 @@ func (va *ValidationAuthorityImpl) validateDNS01(identifier core.AcmeIdentifier,
 	for _, element := range txts {
 		if subtle.ConstantTimeCompare([]byte(element), []byte(authorizedKeysDigest)) == 1 {
 			challenge.Status = core.StatusValid
+			challenge.ValidationRecord = []core.ValidationRecord{
+				core.ValidationRecord{
+					Hostname: identifier.Value,
```

Comment:
Also, we talked about the authority field going somewhere. Where's that happening at?